ingestion/src/metadata/clients/domo_client.py

```diff
@@ -34,6 +36,57 @@
 WORKFLOW_URL = "dataprocessing/v1/dataflows"
 
 
+class DomoBaseModel(BaseModel):
+    """
+    Domo basic configurations
+    """
+
+    class Config:
+        extra = Extra.allow
+
+    id: str
+    name: str
+
+
+class DomoOwner(BaseModel):
+    """
+    Owner Owner Details
+    """
+
+    displayName: str
```

Comment:
curiosity: can we get the email out of the request? With that it should be rather easy to ingest ownership to OM

Reply:
Yes , I was actually working on it.